Main/Source/char.cpp

```diff
@@ -8738,6 +8769,12 @@ void character::LeprosyHandler()
   CheckDeath(CONST_S("killed by leprosy"));
 }
 
+void character::VampirismHandler()
+{
+  EditExperience(WISDOM, -25, 1 << 1);
```

Comment:
This would take 100,000 drains to reduce Wisdom from 10 to 9, if my calculations are correct.  I can help you decide which value and speed to assign, if you tell me how fast you want it to change, or you can look [at Dulcis training](https://github.com/Attnam/ivan/pull/86#issuecomment-188586400) and [merchant training](https://github.com/Attnam/ivan/pull/95) for ideas.